frontend/src/components/storage/VolumeList.tsx

```diff
@@ -67,6 +67,11 @@ export default function VolumeList() {
           getter: volume => {
             const claim = volume.spec.claimRef?.name;
             const claimNamespace = volume.spec.claimRef?.namespace;
+
+            if (!claim) {
+              return null;
+            }
```

Comment:
We should return before checking the claimNamespace, as it's unused if claim is empty/undefined anyway...

Reply:
updated